### DISCUSSION TITLE
How do I write a pathway to stimuli in my JPsych script after uploading them on Cognition.run?

### DISCUSSION THREAD

**Discussion by DDiacono** [jspsych/jsPsych#3207] [Q&A]
I uploaded my experiment on cognition.run and uploaded my audio stimuli as mp3 files in the Stimuli bar on the left. But the stimuli in my script are still being sourced from my local files. 

  timeline.push({
    type: jsPsychAudioButtonResponse,
    stimulus: 'sound/moz source.mp3',
    choices: ['Continue'],
    prompt: "<p>Listen to this track. Press 'Continue' when ready.</p>",
    response_allowed_while_playing: false

How do I link the stimuli I uploaded to Cognition.run into the script?

**Comment by Shaobin-Jiang** [comment]
What do you mean by "sourced from my local files"? That is utterly impossible because the page has no way of getting access to your local audio files.
